infrastructure/main.tf

```diff
@@ -21,6 +21,20 @@ module "blob-router-db" {
   subscription       = "${var.subscription}"
 }
 
+module "reform-blob-router-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = "${var.product}-${var.component}"
```

Comment:
I also doubt this terraform module will allow to create this db as the name will clash. We are re-using product, env so this will produce same Azure Database resource there. Well - will try to but since it is existing it will fail so some other name should be used. I suggest to discuss about all this with the team actually